_targets.R

```diff
@@ -74,7 +74,8 @@ list(
     name = locs,
     command = locs_save,
     read = read_csv(!!.x),
-    packages = "readr"
+    packages = "readr",
+    error = "null"
```

Comment:
If no user-provided locations, like in the example for Great Salt Lake where I only had polygons, the pipeline errored.